sentry_sdk/utils.py

```diff
@@ -1096,6 +1119,73 @@ def exc_info_from_error(error):
     return exc_info
 
 
+def get_full_stack():
+    # type: () -> List[Dict[str, Any]]
+    """
+    Returns a serialized representation of the full stack from the first frame that is not in sentry_sdk.
+    """
+    stack_info = []
+
+    # Walk up the stack
+    frame = sys._getframe(1)  # type: Optional[FrameType]
+    while frame:
+        in_sdk = False
+        try:
+            if "sentry_sdk" in frame.f_code.co_filename:
+                in_sdk = True
```

Comment:
might be better to use f_globals['module'] as this is a pretty weak assertion currently